addons/interaction/functions/fnc_getGlassDoor.sqf

```diff
@@ -61,6 +61,6 @@ private _lowestDistance = 0;
 } forEach _doorPos;
 
 // Check if we have a door or if it is the glass part
-if ((isNil "_door") || ((_door find "glass") != -1)) exitWith {};
+if ((isNil "_door") || ((_door find "glass") != -1)) exitWith {}; // line has never worked?
```

Comment:
if `_door` is nil, the exitWith won't fire because `or` is not short circuiting
but as far as I know this func is working fine, so I'd rather not touch it right now